docker-compose.yml

```diff
@@ -18,6 +18,39 @@ services:
       - phabricator.test
       - bmo.test
 
+  lando-ui.test:
+    image: mozilla/landoui
```

Comment:
Well, it is called mozilla/landoui because that is what ops gave us as the DockerHub repo here: https://hub.docker.com/r/mozilla/landoui/. I don't think we can change it to mozilla/lando-ui unless we also change the DockerHub repo which may or may not bother ops. Same for the api. Thoughts @dklawren?